ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts/vlans.py

```diff
@@ -46,16 +46,22 @@ def _parse_adapter_settings(self: EosDesignsFacts, adapter_settings: dict) -> tu
         trunk_groups = set(adapter_settings.get("trunk_groups", []))
         if "vlans" in adapter_settings and adapter_settings["vlans"] not in ["all", "", None]:
             vlans.update(map(int, range_expand(str(adapter_settings["vlans"]))))
```

Comment:
This section might prevent us from detecting the trunk phone mode and no native_vlan.
Would it be more intuitive that we use the `vlans` field for the data vlan instead of the native_vlan? I know EOS uses native_vlan, so for someone knowing eos config they might feel it is more natural to use native vlan. But for others setting vlans would suddenly break things.

Reply:
Addressed in utils for the structured config module.